codecov.yml

```diff
@@ -25,6 +25,7 @@ flags:
   cosmwasm-schema:
     paths:
       - packages/schema/
+      - packages/schema-derive/
```

Comment:
Adding `packages/schema-derive/` files to the `cosmwasm-schema` coverage report feels confusing. I think it's better to introduce a new flag for clarity, even if that has 0 coverage.